dbms/src/Storages/Page/V3/Blob/BlobStat.h

```diff
@@ -118,6 +119,39 @@ class BlobStats
 
     using BlobStatPtr = std::shared_ptr<BlobStat>;
 
+    class BlobFileIdManager
+    {
+    public:
+        // Used at restart to add old blob file id to manager
+        void addFileId(BlobFileId file_id);
+
+        BlobFileId nextFileId(PageType page_type, const std::lock_guard<std::mutex> &);
+
+        static PageType getBlobFileType(BlobFileId file_id);
+
+    private:
+        static inline bool isRaftFileId(BlobFileId file_id)
+        {
+            return file_id >= FIRST_RAFT_FILE_ID;
+        }
```

Comment:
`inline` is useless here?

Reply:
`static bool isRaftFileId(BlobFileId file_id) const`

Reply:
It is a static function. So `const` is not applicable here. And `inline` may be useful here and even it is useless it should do no harm.

Reply:
ok